Added Oracle Databes installation link

christophhagelkruys opened the pull request:
added using docker without sudo

EmmanuelKasper commented:
Thank you for this pull request !
IMHO the docker as non-root user part is a bit overkill here and not needed in the alanda documentation.
Also we need to speicify in the list from components that we need oracle >= 11.2 ( we're installing 18.0 when following the oracle link ) 
Just fix these two points and after that I will merge the Pull Request !

christophhagelkruys commented:
Requirement: ODB 11.2.0 changed to >= 11.2.0
deleted using docker without sudo 
Added one line before "create user alanda identified as alanda;",
because I could not create this user without c## infront of it, without this line.

EmmanuelKasper commented:
Nite tip for avoiding the ugly c##, thanks, merging now.
